src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1350,9 +1350,10 @@ protected override void InvokeDefaultAction(string path)
                     invokeProcess.Start();
                 }
 #elif CORECLR
-                catch (Win32Exception ex) when (ex.NativeErrorCode == 193)
+                catch (Win32Exception ex) when (ex.NativeErrorCode == 193 || ex.NativeErrorCode == 5)
```

Comment:
Can you add `|| ex.NativeErrorCode == 5` to the catch in `Unix` block as well? `ii <folder-path>` doesn't work in Linux, and I believe it's because the `NativeErrorCode` in that case on Unix is also `5` (or a different value).